benchmark_runner/common/oc/oc.py

```diff
@@ -659,7 +659,7 @@ def wait_for_ready(self, label: str, run_type: str = 'pod', workload: str = '',
     @typechecked
     def exec(self, command: str, pod_name: str, namespace: str = environment_variables.environment_variables_dict['namespace'], container: str = ''):
         """
-        oc exec a command and return the answer
+        This method execs a command and return the answer
```

Comment:
execs a command within a specified pod and optional container and returns the output